### PR TITLE
[WNMGDS-159] Move site package guide page into docs site

### PR DESCRIPTION
**Pull request by line47**
### Changed
Moved site package guidance page into docs site

### Fixed
Fixed links to the site package documentation to go to the new location

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,7 +70,7 @@ These scripts can all be run from the root level of the repo:
 
 #### Theme scripts
 
-You can also use the following scripts to [build and preview a theme](/guides/SITE-PACKAGES-AND-THEMES.md):
```
Comment (bernardwang) [inline README.md]: I'm a little confused by this change, why did it change from a relative URL? Is this the only example of a hardcoded link going to another part of the page?

Comment (line47) [reply README.md]: Thanks this should not have changed 👍

Comment (line47) [reply README.md]: This page got moved into the design system doc site and not only on GitHub now 👍

**Comment by line47** [comment]
@phiden can you take look here?

**Pull request merged**